core/src/main/scala/akka/persistence/dynamodb/internal/QueryDao.scala

```diff
@@ -277,7 +317,8 @@ import software.amazon.awssdk.services.dynamodb.model.QueryResponse
               writeTimestamp = getTimestamp(item),
               readTimestamp = InstantFactory.now(),
               payload = None, // lazy loaded for backtracking
-              serId = item.get(EventSerId).n().toInt,
+              serId =
+                if (item.containsKey(EventSerId)) item.get(EventSerId).n().toInt else backtrackingBreadcrumbSerId(),
```

Comment:
We already return `ItemInJournal` for regular queries, which can be either `SerializedJournalItem` or `JournalItemWithBreadcrumb`. Shouldn't we do the same for backtracking?

There's already an `includePayload` flag for `createItemFromJournal`. Looks like backtracking items also exclude the metadata. But could reuse `createItemFromJournal` and produce backtracking version of `JournalItemWithBreadcrumb`.

Reply:
Although I guess we don't want to follow the breadcrumb just to get the serializer id for large events (without payload for backtracking).

Rather than searching for a free id, could be possible to just use 0? Also used for deleted/expired events, and maps to empty payload. Since the serId is not used for backtracking, with a lazy load of the event, we could probably always have serId = 0 for backtracking?